examples/producer.java

```diff
@@ -0,0 +1,48 @@
+import java.util.concurrent.ExecutionException;
+import com.memphis.Headers;
+import com.memphis.Memphis;
+import com.memphis.MemphisConnectError;
+import com.memphis.MemphisError;
+import com.memphis.MemphisHeaderError;
+import com.memphis.MemphisSchemaError;
+import com.memphis.producer.Producer;
+
+public class Main {
+
+    public static void main(String[] args) throws ExecutionException, InterruptedException {
+        try {
+            Memphis memphis = new Memphis();
+            memphis.connect("<memphis-host>", "<application type username>", "<broker-token>").get();
```

Comment:
I want the memphis connect function will return a connection object which will allow users to make actions, please take a reference for it from the node.js SDK

Reply:
Relevant for the consumer as well